### ISSUE TITLE
Nightly crash - heap-buffer-overflow

### ISSUE THREAD

**Issue by bevand10** [FreeRDP/FreeRDP#5710]
**Describe the bug**
Crash - memory error - heap-buffer-overflow

**To Reproduce**
See below.

**Expected behavior**
Xfreerdp to run successfully. Nightly worked last Thursday or Friday (I use it most days except the weekend).

**Screenshots**
```
$ /opt/freerdp-nightly/bin/xfreerdp /u:username /d:domain /v:B1-R0S064421 /w:1600 /h:900 /sound
[09:00:36:572] [23511:23512] [INFO][com.freerdp.client.common.cmdline] - loading channelEx rdpdr
[09:00:36:573] [23511:23512] [INFO][com.freerdp.client.common.cmdline] - loading channelEx rdpsnd
[09:00:36:573] [23511:23512] [INFO][com.freerdp.client.common.cmdline] - loading channelEx cliprdr
[09:00:36:581] [23511:23512] [ERROR][com.freerdp.core.proxy] - No scheme in proxy URI
[09:00:36:581] [23511:23512] [INFO][com.freerdp.core] - freerdp_set_last_error resetting error state
[09:00:36:582] [23511:23512] [INFO][com.freerdp.core] - freerdp_set_last_error resetting error state
[09:00:37:749] [23511:23512] [WARN][com.freerdp.crypto] - Certificate verification failure 'unable to get local issuer certificate (20)' at stack position 0
[09:00:37:749] [23511:23512] [WARN][com.freerdp.crypto] - CN = B1-R0S064421.**.**.**.**
Password: 
=================================================================
==23511==ERROR: AddressSanitizer: heap-buffer-overflow on address 0x607000006182 at pc 0x7f55392e7733 bp 0x7f551e5f3f40 sp 0x7f551e5f36e8
READ of size 128 at 0x607000006182 thread T1
    #0 0x7f55392e7732  (/usr/lib/x86_64-linux-gnu/libasan.so.4+0x79732)
    #1 0x7f55375bda6a  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x147a6a)
    #2 0x7f553762a522  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1b4522)
    #3 0x7f553763db1a  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1c7b1a)
    #4 0x7f553765214e  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1dc14e)
    #5 0x7f553763e947  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1c8947)
    #6 0x7f553762842a  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1b242a)
    #7 0x7f553760acac in freerdp_connect (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x194cac)
    #8 0x55b0f049a9f2  (/opt/freerdp-nightly/bin/xfreerdp+0x5a9f2)
    #9 0x7f5536b95ace  (/opt/freerdp-nightly/bin/../lib/libwinpr2.so.2+0x12aace)
    #10 0x7f553625e6da in start_thread (/lib/x86_64-linux-gnu/libpthread.so.0+0x76da)
    #11 0x7f553679b88e in __clone (/lib/x86_64-linux-gnu/libc.so.6+0x12188e)

0x607000006182 is located 0 bytes to the right of 66-byte region [0x607000006140,0x607000006182)
allocated by thread T1 here:
    #0 0x7f553934cd38 in __interceptor_calloc (/usr/lib/x86_64-linux-gnu/libasan.so.4+0xded38)
    #1 0x7f5536bbd07a in ConvertToUnicode (/opt/freerdp-nightly/bin/../lib/libwinpr2.so.2+0x15207a)
    #2 0x7f55375bceef  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x146eef)
    #3 0x7f553762a522  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1b4522)
    #4 0x7f553763db1a  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1c7b1a)
    #5 0x7f553765214e  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1dc14e)
    #6 0x7f553763e947  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1c8947)
    #7 0x7f553762842a  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1b242a)
    #8 0x7f553760acac in freerdp_connect (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x194cac)
    #9 0x55b0f049a9f2  (/opt/freerdp-nightly/bin/xfreerdp+0x5a9f2)
    #10 0x7f5536b95ace  (/opt/freerdp-nightly/bin/../lib/libwinpr2.so.2+0x12aace)
    #11 0x7f553625e6da in start_thread (/lib/x86_64-linux-gnu/libpthread.so.0+0x76da)

Thread T1 created by T0 here:
    #0 0x7f55392a5d2f in __interceptor_pthread_create (/usr/lib/x86_64-linux-gnu/libasan.so.4+0x37d2f)
    #1 0x7f5536b95564  (/opt/freerdp-nightly/bin/../lib/libwinpr2.so.2+0x12a564)
    #2 0x7f5536b95daf in CreateThread (/opt/freerdp-nightly/bin/../lib/libwinpr2.so.2+0x12adaf)
    #3 0x55b0f0495ebf  (/opt/freerdp-nightly/bin/xfreerdp+0x55ebf)
    #4 0x55b0f0453613  (/opt/freerdp-nightly/bin/xfreerdp+0x13613)
    #5 0x7f553669bb96 in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x21b96)

SUMMARY: AddressSanitizer: heap-buffer-overflow (/usr/lib/x86_64-linux-gnu/libasan.so.4+0x79732) 
Shadow bytes around the buggy address:
  0x0c0e7fff8be0: fa fa fd fd fd fd fd fd fd fd fd fd fa fa fa fa
  0x0c0e7fff8bf0: fd fd fd fd fd fd fd fd fd fd fa fa fa fa fd fd
  0x0c0e7fff8c00: fd fd fd fd fd fd fd fd fa fa fa fa 00 00 00 00
  0x0c0e7fff8c10: 00 00 00 00 00 fa fa fa fa fa 00 00 00 00 00 00
  0x0c0e7fff8c20: 00 00 00 fa fa fa fa fa 00 00 00 00 00 00 00 00
=>0x0c0e7fff8c30:[02]fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c0e7fff8c40: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c0e7fff8c50: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c0e7fff8c60: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c0e7fff8c70: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c0e7fff8c80: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
Shadow byte legend (one shadow byte represents 8 application bytes):
  Addressable:           00
  Partially addressable: 01 02 03 04 05 06 07 
  Heap left redzone:       fa
  Freed heap region:       fd
  Stack left redzone:      f1
  Stack mid redzone:       f2
  Stack right redzone:     f3
  Stack after return:      f5
  Stack use after scope:   f8
  Global redzone:          f9
  Global init order:       f6
  Poisoned by user:        f7
  Container overflow:      fc
  Array cookie:            ac
  Intra object redzone:    bb
  ASan internal:           fe
  Left alloca redzone:     ca
  Right alloca redzone:    cb
==23511==ABORTING
```

**Application details**
```
$ /opt/freerdp-nightly/bin/xfreerdp -version
This is FreeRDP version 2.0.0-dev5 (02614cce4)
```

**Command line used**
```
$ /opt/freerdp-nightly/bin/xfreerdp /u:username /d:domain /v:target_win10_vm /w:1600 /h:900 /sound
```

**Output of `/buildconfig`**
```
$ /opt/freerdp-nightly/bin/xfreerdp /buildconfig
This is FreeRDP version 2.0.0-dev5 (02614cce4)
Build configuration: BUILD_TESTING=OFF BUILTIN_CHANNELS=ON HAVE_AIO_H=1 HAVE_EXECINFO_H=1 HAVE_FCNTL_H=1 HAVE_INTTYPES_H=1 HAVE_MATH_C99_LONG_DOUBLE=1 HAVE_POLL_H=1 HAVE_PTHREAD_MUTEX_TIMEDLOCK=ON HAVE_PTHREAD_MUTEX_TIMEDLOCK_LIB=1 HAVE_PTHREAD_MUTEX_TIMEDLOCK_SYMBOL= HAVE_SYSLOG_H=1 HAVE_SYS_EVENTFD_H=1 HAVE_SYS_FILIO_H= HAVE_SYS_MODEM_H= HAVE_SYS_SELECT_H=1 HAVE_SYS_SOCKIO_H= HAVE_SYS_STRTIO_H= HAVE_SYS_TIMERFD_H=1 HAVE_TM_GMTOFF=1 HAVE_UNISTD_H=1 HAVE_XI_TOUCH_CLASS=1 WITH_ALSA=ON WITH_CAIRO=ON WITH_CCACHE=ON WITH_CHANNELS=ON WITH_CLANG_FORMAT=ON WITH_CLIENT=ON WITH_CLIENT_AVAILABLE=1 WITH_CLIENT_CHANNELS=ON WITH_CLIENT_CHANNELS_AVAILABLE=1 WITH_CLIENT_COMMON=ON WITH_CLIENT_INTERFACE=OFF WITH_CUPS=ON WITH_DEBUG_ALL=OFF WITH_DEBUG_CAPABILITIES=OFF WITH_DEBUG_CERTIFICATE=OFF WITH_DEBUG_CHANNELS=OFF WITH_DEBUG_CLIPRDR=OFF WITH_DEBUG_DVC=OFF WITH_DEBUG_KBD=OFF WITH_DEBUG_LICENSE=OFF WITH_DEBUG_MUTEX=OFF WITH_DEBUG_NEGO=OFF WITH_DEBUG_NLA=OFF WITH_DEBUG_NTLM=OFF WITH_DEBUG_RAIL=OFF WITH_DEBUG_RDP=OFF WITH_DEBUG_RDPDR=OFF WITH_DEBUG_RDPEI=OFF WITH_DEBUG_RDPGFX=OFF WITH_DEBUG_REDIR=OFF WITH_DEBUG_RFX=OFF WITH_DEBUG_RINGBUFFER=OFF WITH_DEBUG_SCARD=OFF WITH_DEBUG_SND=OFF WITH_DEBUG_SVC=OFF WITH_DEBUG_SYMBOLS=OFF WITH_DEBUG_THREADS=OFF WITH_DEBUG_TIMEZONE=OFF WITH_DEBUG_TRANSPORT=OFF WITH_DEBUG_TSG=OFF WITH_DEBUG_TSMF=OFF WITH_DEBUG_WND=OFF WITH_DEBUG_X11=OFF WITH_DEBUG_X11_CLIPRDR=OFF WITH_DEBUG_X11_LOCAL_MOVESIZE=OFF WITH_DEBUG_XV=OFF WITH_DSP_EXPERIMENTAL=OFF WITH_DSP_FFMPEG=ON WITH_EVENTFD_READ_WRITE=1 WITH_FAAC=OFF WITH_FAAD2=OFF WITH_FFMPEG=TRUE WITH_FFMPEG=TRUE WITH_GFX_H264=ON WITH_GPROF=OFF WITH_GSM=OFF WITH_GSSAPI=OFF WITH_GSTREAMER_0_10=ON WITH_GSTREAMER_1_0=ON WITH_ICU=OFF WITH_IPP=OFF WITH_JPEG=OFF WITH_LAME=OFF WITH_LIBRARY_VERSIONING=ON WITH_LIBSYSTEMD=OFF WITH_MACAUDIO=OFF WITH_MACAUDIO=OFF WITH_MACAUDIO_AVAILABLE=0 WITH_MANPAGES=ON WITH_MBEDTLS=OFF WITH_OPENH264=OFF WITH_OPENSLES=OFF WITH_OPENSSL=ON WITH_OSS=ON WITH_PAM=ON WITH_PCSC=ON WITH_PROFILER=OFF WITH_PULSE=ON WITH_SAMPLE=OFF WITH_SANITIZE_ADDRESS=ON WITH_SANITIZE_ADDRESS_AVAILABLE=1 WITH_SANITIZE_MEMORY=OFF WITH_SANITIZE_MEMORY=OFF WITH_SANITIZE_MEMORY_AVAILABLE=0 WITH_SANITIZE_THREAD=OFF WITH_SANITIZE_THREAD=OFF WITH_SANITIZE_THREAD_AVAILABLE=0 WITH_SERVER=ON WITH_SERVER_CHANNELS=ON WITH_SERVER_INTERFACE=ON WITH_SMARTCARD_INSPECT=OFF WITH_SOXR=OFF WITH_SSE2=ON WITH_SWSCALE=OFF WITH_THIRD_PARTY=OFF WITH_VALGRIND_MEMCHECK=OFF WITH_VALGRIND_MEMCHECK=OFF WITH_VALGRIND_MEMCHECK_AVAILABLE=0 WITH_WAYLAND=OFF WITH_WINPR_TOOLS=ON WITH_X11=ON WITH_X264=OFF WITH_XCURSOR=ON WITH_XDAMAGE=ON WITH_XEXT=ON WITH_XFIXES=ON WITH_XI=ON WITH_XINERAMA=ON WITH_XKBFILE=ON WITH_XRANDR=ON WITH_XRENDER=ON WITH_XSHM=ON WITH_XTEST=ON WITH_XV=ON WITH_ZLIB=ON
Build type:          RelWithDebInfo
CFLAGS:              -g -O2 -fdebug-prefix-map=/build/freerdp-nightly-2.0.0+0~20191108024828.722~1.gbp02614c=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -Wall -Wno-unused-result -Wno-unused-but-set-variable -Wno-deprecated-declarations -fvisibility=hidden -Wimplicit-function-declaration -Wredundant-decls -g -fsanitize=address -fsanitize-address-use-after-scope -fno-omit-frame-pointer -DWINPR_DLL
Compiler:            GNU, 7.3.0
Target architecture: x64
```
```
$ cat /etc/apt/sources.list.d/freerdp.list
deb http://pub.freerdp.com/repositories/deb/bionic/ freerdp-nightly main
```

**OS version connecting to**
Windows 10

**OS version connecting from**
```
$ lsb_release -a
LSB Version:	core-9.20170808ubuntu1-noarch:printing-9.20170808ubuntu1-noarch:security-9.20170808ubuntu1-noarch
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.3 LTS
Release:	18.04
Codename:	bionic
$ uname -r
5.3.0-19-generic
```

**If available the log output from a run with `/log-level:trace`**
```
$ win10r /log-level:trace
[09:08:03:307] [23753:23754] [INFO][com.freerdp.client.common.cmdline] - loading channelEx rdpdr
[09:08:03:307] [23753:23754] [INFO][com.freerdp.client.common.cmdline] - loading channelEx rdpsnd
[09:08:03:307] [23753:23754] [DEBUG][com.freerdp.channels.cliprdr.client] - VirtualChannelEntryEx
[09:08:03:307] [23753:23754] [INFO][com.freerdp.client.common.cmdline] - loading channelEx cliprdr
[09:08:03:309] [23753:23754] [DEBUG][com.freerdp.client.x11] - Searching for XInput pointer device
[09:08:03:309] [23753:23754] [DEBUG][com.freerdp.client.x11] - Pointer device: 11
[09:08:03:316] [23753:23754] [DEBUG][com.freerdp.core.nego] - Enabling security layer negotiation: TRUE
[09:08:03:316] [23753:23754] [DEBUG][com.freerdp.core.nego] - Enabling restricted admin mode: FALSE
[09:08:03:316] [23753:23754] [DEBUG][com.freerdp.core.nego] - Enabling RDP security: TRUE
[09:08:03:316] [23753:23754] [DEBUG][com.freerdp.core.nego] - Enabling TLS security: TRUE
[09:08:03:316] [23753:23754] [DEBUG][com.freerdp.core.nego] - Enabling NLA security: TRUE
[09:08:03:316] [23753:23754] [DEBUG][com.freerdp.core.nego] - Enabling NLA extended security: FALSE
[09:08:03:316] [23753:23754] [DEBUG][com.freerdp.core.nego] - state: NEGO_STATE_NLA
[09:08:03:316] [23753:23754] [DEBUG][com.freerdp.core.nego] - Attempting NLA security
[09:08:03:316] [23753:23754] [ERROR][com.freerdp.core.proxy] - No scheme in proxy URI
[09:08:03:316] [23753:23754] [INFO][com.freerdp.core] - freerdp_set_last_error resetting error state
[09:08:03:316] [23753:23754] [INFO][com.freerdp.core] - freerdp_set_last_error resetting error state
[09:08:03:316] [23753:23754] [DEBUG][com.freerdp.core] - connecting to peer 127.0.0.1
[09:08:03:317] [23753:23754] [DEBUG][com.freerdp.core.nego] - RequestedProtocols: 3
[09:08:03:383] [23753:23754] [DEBUG][com.freerdp.core.nego] - RDP_NEG_RSP
[09:08:03:383] [23753:23754] [DEBUG][com.freerdp.core.nego] - selected_protocol: 2
[09:08:03:383] [23753:23754] [DEBUG][com.freerdp.core.nego] - state: NEGO_STATE_FINAL
[09:08:03:383] [23753:23754] [DEBUG][com.freerdp.core.nego] - Negotiated NLA security
[09:08:03:383] [23753:23754] [DEBUG][com.freerdp.core.nego] - nego_security_connect with PROTOCOL_HYBRID
[09:08:03:505] [23753:23754] [WARN][com.freerdp.crypto] - Certificate verification failure 'unable to get local issuer certificate (20)' at stack position 0
[09:08:03:505] [23753:23754] [WARN][com.freerdp.crypto] - CN = B1-R0S064421.**.**.**
[09:08:03:506] [23753:23754] [DEBUG][com.winpr.utils] - Could not open SAM file!
Password: 
[09:08:08:137] [23753:23754] [DEBUG][com.winpr.sspi] - InitSecurityInterfaceExA
[09:08:08:138] [23753:23754] [DEBUG][com.freerdp.core.nla] - nla_client_init 400 : packageName=Negotiate ; cbMaxToken=12256
[09:08:08:138] [23753:23754] [TRACE][com.freerdp.core.nla] -  InitializeSecurityContext status SEC_I_CONTINUE_NEEDED [0x00090312]
[09:08:08:138] [23753:23754] [DEBUG][com.freerdp.core.nla] - Sending Authentication Token
[09:08:08:161] [23753:23754] [DEBUG][com.freerdp.core.nla] - CredSSP protocol support 6, peer supports 6
[09:08:08:162] [23753:23754] [TRACE][com.freerdp.core.nla] - InitializeSecurityContext  SEC_I_COMPLETE_NEEDED [0x00090313]
[09:08:08:162] [23753:23754] [DEBUG][com.freerdp.core.nla] - Sending Authentication Token
[09:08:08:232] [23753:23754] [DEBUG][com.freerdp.core.gcc] - Server rdp encryption method: NONE
[09:08:08:348] [23753:23754] [DEBUG][com.freerdp.core.info] - Client Info Packet Flags = INFO_MOUSE|INFO_DISABLECTRLALTDEL|INFO_UNICODE|INFO_MAXIMIZESHELL|INFO_LOGONNOTIFY|INFO_COMPRESSION|INFO_ENABLEWINDOWSKEY|INFO_FORCE_ENCRYPTED_CS_PDU|INFO_LOGONERRORS|INFO_MOUSE_HAS_WHEEL
=================================================================
==23753==ERROR: AddressSanitizer: heap-buffer-overflow on address 0x6070000061f2 at pc 0x7f43b68c4733 bp 0x7f439bbf3f40 sp 0x7f439bbf36e8
READ of size 128 at 0x6070000061f2 thread T1
    #0 0x7f43b68c4732  (/usr/lib/x86_64-linux-gnu/libasan.so.4+0x79732)
    #1 0x7f43b4b9aa6a  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x147a6a)
    #2 0x7f43b4c07522  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1b4522)
    #3 0x7f43b4c1ab1a  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1c7b1a)
    #4 0x7f43b4c2f14e  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1dc14e)
    #5 0x7f43b4c1b947  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1c8947)
    #6 0x7f43b4c0542a  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1b242a)
    #7 0x7f43b4be7cac in freerdp_connect (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x194cac)
    #8 0x55966a93b9f2  (/opt/freerdp-nightly/bin/xfreerdp+0x5a9f2)
    #9 0x7f43b4172ace  (/opt/freerdp-nightly/bin/../lib/libwinpr2.so.2+0x12aace)
    #10 0x7f43b383b6da in start_thread (/lib/x86_64-linux-gnu/libpthread.so.0+0x76da)
    #11 0x7f43b3d7888e in __clone (/lib/x86_64-linux-gnu/libc.so.6+0x12188e)

0x6070000061f2 is located 0 bytes to the right of 66-byte region [0x6070000061b0,0x6070000061f2)
allocated by thread T1 here:
    #0 0x7f43b6929d38 in __interceptor_calloc (/usr/lib/x86_64-linux-gnu/libasan.so.4+0xded38)
    #1 0x7f43b419a07a in ConvertToUnicode (/opt/freerdp-nightly/bin/../lib/libwinpr2.so.2+0x15207a)
    #2 0x7f43b4b99eef  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x146eef)
    #3 0x7f43b4c07522  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1b4522)
    #4 0x7f43b4c1ab1a  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1c7b1a)
    #5 0x7f43b4c2f14e  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1dc14e)
    #6 0x7f43b4c1b947  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1c8947)
    #7 0x7f43b4c0542a  (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x1b242a)
    #8 0x7f43b4be7cac in freerdp_connect (/opt/freerdp-nightly/bin/../lib/libfreerdp2.so.2+0x194cac)
    #9 0x55966a93b9f2  (/opt/freerdp-nightly/bin/xfreerdp+0x5a9f2)
    #10 0x7f43b4172ace  (/opt/freerdp-nightly/bin/../lib/libwinpr2.so.2+0x12aace)
    #11 0x7f43b383b6da in start_thread (/lib/x86_64-linux-gnu/libpthread.so.0+0x76da)

Thread T1 created by T0 here:
    #0 0x7f43b6882d2f in __interceptor_pthread_create (/usr/lib/x86_64-linux-gnu/libasan.so.4+0x37d2f)
    #1 0x7f43b4172564  (/opt/freerdp-nightly/bin/../lib/libwinpr2.so.2+0x12a564)
    #2 0x7f43b4172daf in CreateThread (/opt/freerdp-nightly/bin/../lib/libwinpr2.so.2+0x12adaf)
    #3 0x55966a936ebf  (/opt/freerdp-nightly/bin/xfreerdp+0x55ebf)
    #4 0x55966a8f4613  (/opt/freerdp-nightly/bin/xfreerdp+0x13613)
    #5 0x7f43b3c78b96 in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x21b96)

SUMMARY: AddressSanitizer: heap-buffer-overflow (/usr/lib/x86_64-linux-gnu/libasan.so.4+0x79732) 
Shadow bytes around the buggy address:
  0x0c0e7fff8be0: fa fa fd fd fd fd fd fd fd fd fd fd fa fa fa fa
  0x0c0e7fff8bf0: fd fd fd fd fd fd fd fd fd fd fa fa fa fa fd fd
  0x0c0e7fff8c00: fd fd fd fd fd fd fd fd fa fa fa fa fd fd fd fd
  0x0c0e7fff8c10: fd fd fd fd fd fd fa fa fa fa 00 00 00 00 00 00
  0x0c0e7fff8c20: 00 00 00 fa fa fa fa fa 00 00 00 00 00 00 00 00
=>0x0c0e7fff8c30: 00 fa fa fa fa fa 00 00 00 00 00 00 00 00[02]fa
  0x0c0e7fff8c40: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c0e7fff8c50: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c0e7fff8c60: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c0e7fff8c70: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c0e7fff8c80: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
Shadow byte legend (one shadow byte represents 8 application bytes):
  Addressable:           00
  Partially addressable: 01 02 03 04 05 06 07 
  Heap left redzone:       fa
  Freed heap region:       fd
  Stack left redzone:      f1
  Stack mid redzone:       f2
  Stack right redzone:     f3
  Stack after return:      f5
  Stack use after scope:   f8
  Global redzone:          f9
  Global init order:       f6
  Poisoned by user:        f7
  Container overflow:      fc
  Array cookie:            ac
  Intra object redzone:    bb
  ASan internal:           fe
  Left alloca redzone:     ca
  Right alloca redzone:    cb
==23753==ABORTING
$
```

**Comment by akallabeth** [comment]
Fixed with #5709

**Comment by bevand10** [comment]
I can confirm that today, nightly (installed a few moments ago)
```
$ /opt/freerdp-nightly/bin/freerdp -version
This is FreeRDP version 2.0.0-dev5 (8cf19fd94)
```
does not crash when connecting to the same target host as above.

Many thanks!
--D.
